bgpd/bgp_attr.c

```diff
@@ -2804,9 +2803,8 @@ static void bgp_packet_mpattr_tea(struct bgp *bgp, struct peer *peer,
 
 	if (attrlenfield > 0xff) {
 		/* 2-octet length field */
-		stream_putc(s,
-			    BGP_ATTR_FLAG_TRANS | BGP_ATTR_FLAG_OPTIONAL
-				    | BGP_ATTR_FLAG_EXTLEN);
+		stream_putc(s, BGP_ATTR_FLAG_TRANS | BGP_ATTR_FLAG_OPTIONAL
+				       | BGP_ATTR_FLAG_EXTLEN);
```

Comment:
You do realize that the tools we used to convert patches forward to the new style rewrote stuff?